.azure/pipelines/jobs/default-build.yml

```diff
@@ -101,14 +101,14 @@ jobs:
         vmImage: macOS-10.15
       ${{ if eq(parameters.agentOs, 'Linux') }}:
         ${{ if eq(parameters.useHostedUbuntu, true) }}:
-          vmImage: ubuntu-16.04
+          vmImage: ubuntu-18.04
         ${{ if eq(parameters.useHostedUbuntu, false) }}:
           ${{ if eq(variables['System.TeamProject'], 'public') }}:
             name: NetCorePublic-Pool
-            queue: BuildPool.Ubuntu.1604.Amd64.Open
```

Comment:
@sebastienros should work if you undo this change and the change at line 111 (keep line 104)

Reply:
Actually nvm I'll do it now